src/scala/microsoft-spark-2-3/src/main/scala/org/apache/spark/api/dotnet/DotnetBackend.scala

```diff
@@ -30,6 +30,10 @@ class DotnetBackend extends Logging {
   private[this] var channelFuture: ChannelFuture = _
   private[this] var bootstrap: ServerBootstrap = _
   private[this] var bossGroup: EventLoopGroup = _
+  private[this] val objectsTracker = new JVMObjectTracker
```

Comment:
nit: `objectTracker` to match the class name?

Reply:
Agree. Field `objectsTracker` was renamed to `objectTracker `